src/applications/personalization/rated-disabilities/tests/e2e/ratedDisabilitiesError.e2e.spec.js

```diff
@@ -6,31 +6,67 @@ const Mock = require('platform/testing/e2e/mock-helpers');
 const Timeouts = require('platform/testing/e2e/timeouts');
 
 function runRatedDisabilitiesTest(browser) {
-  browser.assert.containsText(
-    '.usa-alert-heading',
-    'We’re sorry. Something went wrong on our end',
+  // Because error states for total rating and list are separate but contain identical classes,
+  // a list of elements needs to be collected and their values checked individually.
+  // Though the value checks are nested, they are still evaluated and pass correctly.
+  browser.elements(
+    'css selector',
+    'h2.vads-u-margin-y--0.vads-u-font-size--lg',
+    result => {
+      browser.assert.equal(result.value.length, 2);
+      const totalRatingErrorHeader = result.value[0];
+      const listErrorHeader = result.value[1];
+      browser.elementIdText(totalRatingErrorHeader.ELEMENT, el => {
+        browser
+          .expect(el.value)
+          .to.equal('We don’t have a disability rating on file for you');
+      });
+      browser.elementIdText(listErrorHeader.ELEMENT, el => {
+        browser
+          .expect(el.value)
+          .to.equal('We don’t have rated disabilities on file for you');
+      });
+    },
```

Comment:
This feels sort of wonky to me, but I wasn't able to come up with a better way to make this work. Attempting to select based off `nth-of-type` didn't work for some reason. I checked to make sure the two nested checks get evaluated and they do, but it's kind of weird. They pass silently (don't show up in console when running tests) but if their values are changed then they fail and throw an error...so that's good?